app/library/api_schemas.py

```diff
@@ -45,10 +45,20 @@
         "publish_link": {"type": "string", "pattern": r"^[\S]{1,50}$"},
         "preview_link": {"type": "string", "pattern": r"^[\S]{1,50}$"},
         "expected_solve_time": {"type": "string", "pattern": r"^[a-zA-Z0-9 _-]{,50}$"},
+        "introduction_data": {
+            "type": "object",
+            "properties": {
+                "header_text": {"type": "string"},
```

Comment:
Do we also want a paragraph text for intro? Or just conclusion?

Reply:
I imagined the intro being just a title and a play button. What do you think?

Reply:
I'm good with this for now, but we may want to add in the future.